extensions/gdx-jnigen/src/com/badlogic/gdx/jnigen/BuildTarget.java

```diff
@@ -91,8 +95,20 @@ public BuildTarget (BuildTarget.TargetOs targetType, boolean is64Bit, String[] c
 		this.libraries = "";
 	}
 
+	/** Legacy compatible addition to specify ARM values */
+	public BuildTarget setARM (boolean isARM, String abi) {

```

Comment:
Couldn't this be private/protected?
My quick look didn't show any use outside this class.

Reply:
BuildTarget itself is public, and you could want to extend with additional abi's in the future before libgdx necessarily adds them. I would keep it public.